subsys/bluetooth/controller/ll_sw/ull_chan.c

```diff
@@ -38,8 +39,11 @@
 static uint8_t map[5] = {0xFF, 0xFF, 0xFF, 0xFF, 0x1F};
 static uint8_t count = 37U;
 
+#if defined(CONFIG_BT_LL_SW_SPLIT_LLCP_LEGACY)
 static void chan_map_set(uint8_t const *const chan_map);
```

Comment:
Why not remove the static version in the legacy build and simply use ull_chan_map_set?